### PR TITLE
Explore ticks

### PR DESCRIPTION
**Pull request by msabvid**
Change UniswapDummyAgent to explore a wide range of ticks, instead of manually calibrating drift and vol of gbm.

### CODE REVIEW

File: uniswap_example.py [hunk 1]
```diff
@@ -18,17 +18,39 @@
...
         cache_json = json.load(f)
 
-    cache = verbs.utils.cache_from_json(cache_json)
+    if args.cache:
+        assert (
+            args.alchemy_key is not None
+        ), "Alchemy key required, set with '--alchemy_key' argument"
+        cache = sim.init_cache(
+            args.alchemy_key,
+            19163600,
```
Comment (zombie-einstein) [inline uniswap_example.py:48]: Do we want to make the block number configurable?

File: verbs_examples/agents/uniswap_agent.py [hunk 2]
```diff
@@ -854,11 +845,60 @@ def update(self, rng: np.random.Generator, env) -> List[verbs.types.Transaction]
         list
             List of transactions to be processed in the next block
             of the simulation. This can be an empty list if the
-            agent is not submitting any transactions.
+            agent is not submitting any transacti
```
Comment (zombie-einstein) [inline verbs_examples/agents/uniswap_agent.py:848]: docstring seems to be cut off

File: verbs_examples/uniswap/sim.py [hunk 3]
```diff
@@ -157,8 +157,6 @@ def init_cache(
     block_number: int,
     seed: int,
     n_steps: int,
-    mu: float = 0.1,
-    sigma: float = 0.6,
```
Comment (zombie-einstein) [inline verbs_examples/uniswap/sim.py:161]: How come these are not needed now?

Comment (msabvid) [reply verbs_examples/uniswap/sim.py:161]: The DummyUniswapAgent is exploring a range of ticks one by one, so in the initial simulation the Gbm is not involved and mu and sigma are not needed.

File: uniswap_example.py (hunk 1 above)
Comment (msabvid) [reply uniswap_example.py]: added in a new commit

File: verbs_examples/agents/uniswap_agent.py (hunk 2 above)
Comment (msabvid) [reply verbs_examples/agents/uniswap_agent.py]: fixed in new commit

**Review by zombie-einstein — APPROVED** [review]

**Pull request merged**